egs/wsj/s5/utils/subword/prepare_subword_text.sh

```diff
@@ -0,0 +1,42 @@
+#!/bin/bash
+
+# 2019 Dongji Gao
+
+# This script generate subword text form word text.
```

Comment:
generates
also maybe give a bit more description and talk about input output format